web/object.html

```diff
@@ -142,6 +142,10 @@ <h4>Descriptive Information</h4>
                     </select>
                     <!-- TODO -->
                     <!--<h3> Upload Media </h3>-->
+                    <label>Link a photograph or depiction: </label>
+                    <input type="url" deer-key="depiction" >

```

Comment:
Not sure `depiction` is in the schema.  Do [image](https://schema.org/image) as the deer-key.

Reply:
You're right. I had `FOAF` on the mind.